### DISCUSSION TITLE
v6.2.3

### DISCUSSION THREAD

**Discussion by fcakyon** [fcakyon/yolov5-pip#165] [Announcements]
## What's Changed
* update to ultralytics/yolov5 26.10.22 by @fcakyon in https://github.com/fcakyon/yolov5-pip/pull/164

**Full Changelog**: https://github.com/fcakyon/yolov5-pip/compare/6.2.2...6.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/fcakyon/yolov5-pip/releases/tag/6.2.3'>v6.2.3</a>.</em>

**Comment by RicardasNaujokas** [comment]
Hi @fcakyon,

when trying to use DetectMultiBackend class from yolov5.models.common it throws module not found error: "No module named 'models.yolo'".

I suspect that the error is caused when class is trying to invoke "attempt_load" from yolov5.models.experimental.

See yolov5\models\experimental.py line 75: from models.yolo import Detect, Model

I think that it needs to be changed to -> from yolov5.models.yolo import Detect, Model

**Reply by fcakyon** [reply]
this line gives me no error: `from yolov5.models.common import DetectMultiBackend`

**Reply by fcakyon** [reply]
have you properly installed the `yolov5==6.2.3`?

**Reply by RicardasNaujokas** [reply]
@fcakyon,

Yes, just importing will not raise the error. 
But when creating the object for example: 
                 model = DetectMultiBackend("customModel.pt");
in case weights is Pytorch you should get the error.

If you take a look at the DetectMultiBackend class you will find line (345):
        model = attempt_load(weights if isinstance(weights, list) else w, device=device, inplace=True, fuse=fuse)
![image](https://user-images.githubusercontent.com/98582836/198564474-4ca7d732-427c-40eb-a5cd-31340b5b4dd6.png)

This line is invoking  attempt_load method, which is in experimental.py.
The line 75 in the experimental.py file I think it it causing the error:
          from models.yolo import Detect, Model
![image](https://user-images.githubusercontent.com/98582836/198564931-af5e9dcc-9bdd-422e-8aaf-c6110673b6dc.png)

And package do not contains model.yolo ... in the package it is under yolov5.models.yolo
